Bottom: 8899349b7aa7c754234f8de8f19686fcb4234c3c
Top:    d5c53064b65607ba84eebf6f3b4f58c7377d22a7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-19 00:35:15 +0000

Refresh of add-a-demonstration-model-for-

---

diff --git a/src/hid/common/hidgl_package_acy_resistor.c b/src/hid/common/hidgl_package_acy_resistor.c
index 63e8af3..17eb17c 100644
--- a/src/hid/common/hidgl_package_acy_resistor.c
+++ b/src/hid/common/hidgl_package_acy_resistor.c
@@ -662,7 +662,7 @@ hidgl_draw_acy_resistor (ElementType *element, float surface_depth, float board_
   glRotatef (90, 1., 0., 0.);
 
   /* TEXTURE SETUP */
-  glGetIntegerv (GL_CURRENT_PROGRAM, &restore_sp);
+  glGetIntegerv (GL_CURRENT_PROGRAM, (GLint*)&restore_sp);
   glUseProgram (sp2);
 
   {
